priv/apps/child_chain_api/lib/child_chain_api/api/child_chain.ex

```diff
@@ -0,0 +1,69 @@
+# NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
```

Comment:
thats difficult to do considering this is an umbrella structure...

Reply:
I see. I was thinking about the `priv` prefix, unless you're constrained in the choice there - but you'd have to separate it from your non-generated code on this level, so you'd have `priv/` and `generated/` or `priv/generated`

If not, then maybe `child_chain_api` could become sth like `swagger_child_chain_client` or `generated_child_chain_client` or sth? Don't know the constraints, so I don't know if it makes any sense at all.

Reply:
swagger generates an application (with `mix.exs` as well). This is important because it limits you what name can you give to the parent directory of the app (umbrella applications in `mix.exs` needs to match the directory name). though what we will do is that the umbrella application will not have these apps at all (they won't be committed to git at all, but generated in circleci. _you'll be able to generate them when you write new tests!_

Reply:
> though what we will do is that the umbrella application will not have these apps at all

ah, this sounds good then.